python/taichi/_main.py

```diff
@@ -135,6 +136,85 @@ def support_choice_with_dot_py(choice):
 
         return support_choice_with_dot_py
 
+    @register
+    def gallery(self, argumets: list = sys.argv[2:]):
```

Comment:
Nit:add a docstring for gallery as well?

Reply:
Sure. Docstring added.